.golangci.yml

```diff
@@ -15,24 +13,17 @@ linters:
   # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
   disable-all: true
   enable:
-    - deadcode
     - depguard
     - dogsled
     - gofmt
     - goimports
     - goprintffuncname
-    - gosimple
     - govet
     - ineffassign
     - misspell
     - rowserrcheck
-    - staticcheck
```

Comment:
Hmm that's the linter I would not drop

Reply:
Since we updated it to the last version, there are a lot of issues that were ignored. I'm dropping it only to make the 1.19 update possible and then work on https://github.com/kubernetes-sigs/external-dns/issues/3194 to reintroduce that and the other ones.

Reply:
Ok